.grype.yml

```diff
@@ -0,0 +1,23 @@
+ignore:
```

Comment:
Could we add a comment block at the top of the file explaining what this file is, what it is used for, and link to other relevant docs that say more (e.g. vulnerability-management.md)

Reply:
Yeap, I'm going to change how this is set in the next commit. I cleaned it up in our repo and will pull the changes over